demoConfig.csv

```diff
@@ -1,11 +1,11 @@
-DevTestLabName,ResourceGroupName,IpConfig,BastionEnabled,SharedImageGalleryName,ShutDownTime,TimezoneId,LabRegion,LabOwners,LabUsers
-Cybi1, Cybe3r,Private,true,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi2, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi3, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi4, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi5, Cybe3r,Private,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi6, Cybe3r,Private,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi7, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi8, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi9, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi10, Cybe3r1900,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
+DevTestLabName,ResourceGroupName,VmCreationResourceGroupName,IpConfig,BastionEnabled,SharedImageGalleryName,ShutDownTime,TimezoneId,LabRegion,LabOwners,LabUsers
+Cybi1, Cybe3r, default,Private,true,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
+Cybi2, Cybe3r, default,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
+Cybi3, Cybe3r, default,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
```

Comment:
Could you update the sample CSV to have each lab in their own resource group?  It's a best practice so we don't run out of resource types in the same RG...